src/core/biblio-db.js

```diff
@@ -143,22 +110,13 @@ export const biblioDB = {
       throw new TypeError("id is required");
     }
     const db = await this.ready;
-    return new Promise((resolve, reject) => {
-      const objectStore = db
-        .transaction("alias", "readonly")
-        .objectStore("alias");
-      const range = IDBKeyRange.only(id);
-      const request = objectStore.openCursor(range);
-      request.onsuccess = () => {
-        if (request.result === null) {
-          return resolve(null);
-        }
-        resolve(request.result.value.aliasOf);
-      };
-      request.onerror = () => {
-        reject(new DOMException(request.error.message, request.error.name));
-      };
-    });
+
+    const objectStore = db
+      .transaction("alias", "readonly")
+      .objectStore("alias");
+    const range = IDBKeyRange.only(id);
+    const request = await objectStore.openCursor(range);
```

Comment:
How about `result`, because it's not really a request object anymore?

Reply:
My bad, leaving constant names intact wasn't a very good decision. Will make these changes